src/mfa/mfa.ts

```diff
@@ -0,0 +1,35 @@
+import { WorkOS } from '../workos';
+import { ChallengeFactorOptions } from './interfaces/challenge-factor-options';
+import { Challenge } from './interfaces/challenge.interface';
+import { EnrollFactorOptions } from './interfaces/enroll-factor-options';
+import { Factor } from './interfaces/factor.interface';
+import { VerifyFactorOptions } from './interfaces/verify-factor-options';
+import { VerifyResponse } from './interfaces/verify-factor-response';
+
+export class MFA {
```

Comment:
nit: Can we change the casing to be `Mfa`?

```suggestion
export class Mfa {
```